lib/whitehall/asset_manager_storage.rb

```diff
@@ -44,5 +44,17 @@ def path
     def content_type
       MIME::Types.type_for(filename).first.to_s
     end
+
+    def size
+      response = Services.asset_manager.whitehall_asset(path)
+      if response.has_key?('size') && !response['size'].nil?
+        Integer(response['size'])
+      else
+        false
+      end
+    rescue StandardError => e
```

Comment:
Rescuing `StandardError` feels a bit dangerous. Would it be possible to be a bit more specific?

Reply:
Would rescuing `GdsApi` exceptions be better do you think?

Reply:
Yes, I think so. Is there a single base class exception you can rescue, or failing that a list of specific `GdsApi` exceptions...?

Reply:
When I first saw this, I wondered whether you were also trying to rescue `TypeError` exceptions from`Integer(nil)`, but then I realised you're checking for that in the `if` condition.

Reply:
Yes, its various API errors I'm mostly anticipating. Plus other things that might go wrong given that the schema of the reponse isn't strictly enforced.

Reply:
I'll change this to rescue GdsApi exceptions and their subclasses. Sentry will tell us if we've missed other types.